tf-modules/archive/ems_reporting.tf

```diff
@@ -77,6 +77,8 @@ resource "aws_lambda_function" "ems_product_metadata_report" {
       CollectionsTable    = var.dynamo_tables.collections.name
       cmr_client_id       = var.cmr_client_id
       cmr_provider        = var.cmr_provider
+      cmr_username        = var.cmr_username
```

Comment:
aren't these supposed to come from the secrets manager now? using cmr_password_secret_name?

Reply:
Yeah you are right. I have made the changes in the rec reports to fix this. I did not make the changes here, will do that in the next PR. These use the `CMRSearchConceptQueue`, which I have not updated yet, will do in next PR. Thank you for this.